python/perf-kernels/streamk/tune_streamk.py

```diff
@@ -206,9 +205,9 @@ def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose):
     os.environ['ROCR_VISIBLE_DEVICES'] = str(gpuid)
     jobId = gpuIdx
     while jobId < jobs:
-        kernel_name = get_filename_profile_driver(M, N, K, jobId)
+        kernelname = get_filename_profile_driver(M, N, K, jobId)
         if verbose:
-            print(f"profiling {kernel_name} on GPU {gpuid}")
+            print(f"profiling {kernelname} on GPU {gpuid}")
         run_bash_command_wrapper(
             f"rocprof --stats -o results_{jobId}.csv python {get_filename_profile_driver(M, N, K, jobId)}",
             #            f"rocprofv2 --plugin file --plugin-version 1 --kernel-trace -o {jobId} python {get_filename_profile_driver(M, N, K, jobId)}",
```

Comment:
Can we delete this line?

Reply:
can we keep it here for a while, as currently rocprofv2 has bug that can't return all configs timing, if bug fixed, we could use rocprofv2 may be helpful to reduce profiling costs.

Reply:
Sure. May be add a comment and TODO.